packages/vue-app/template/App.js

```diff
@@ -106,12 +106,17 @@ export default {
   <% } %>
   beforeCreate () {
     Vue.util.defineReactive(this, 'nuxt', this.$options.nuxt)
+    <% if ((features.asyncData || features.fetch) && isFullStatic) { %>
+    if (process.client) {
+      Vue.util.defineReactive(this, 'payloadPath', window.__PAYLOAD_PATH__)
```

Comment:
Side note: It means globalName is not supported

Reply:
Indeed but not needed for this particular feature :)